dist/yaml/ovnkube-master.yaml

```diff
@@ -0,0 +1,181 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master
+  namespace: openshift-sdn
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the OpenShift networking components (kube-proxy, DNS, and ovn-k8s-cni-overlay).
+      It expects that OVS is running on the node.
+    image.openshift.io/triggers: |
```

Comment:
I'd say just remove this annotation for now since it's OpenShift specific.

Reply:
OK